mlflow/tracking/fluent.py

```diff
@@ -69,9 +70,29 @@
     import numpy
     import PIL
 
-_active_run_stack = []
-_active_experiment_id = None
-_last_active_run_id = None
+
+class DefaultSetContextVar:
+    def __init__(self, contextvar, default: Callable):
```

Comment:
Why not simplifying the `__init__` argument to be 
```
def __init__(self, var_name, default: Callable)
```

Reply:
Yep, not sure why I had an issue at some point, contextvars mess up if used within a closure, but in the `__init__` here appears to work fine